### ISSUE TITLE
Some workflow ids breaks the new test site modulo computation

### ISSUE THREAD

**Issue by kptdobe** [adobe/helix-pages#504]
The new module computation magic based on the workflow id does not work in all the cases:
https://app.circleci.com/pipelines/github/adobe/helix-pages/2022/workflows/d6acffb2-d840-4dbf-b63a-f150be2243b8/jobs/7719

Workaround: re-run the tests

cc @trieloff

**Comment by kptdobe** [comment]
This happens if the workflow id contains too many letters.

**Comment by trieloff** [comment]
I'd guess it happens when there are too few letters. If that's the case, a fix would be to add a few ... to the second sed expression.

**Comment by kptdobe** [comment]
Nope it is the other way around: too many letters, after first `sed`, number is small, second `sed` reduces it to nothing.

**Comment by kptdobe** [comment]
Trying to understand the command, I am a bit worried: the workflow id is completely random, thus 2 tests running in parallel may easily end up with the same modulo... The probability was much lower before when we were using the job id because it is sequential...

**Comment by tripodsan** [comment]
what about having a global counter somewhere ?

**Comment by trieloff** [comment]
A global counter won't do unless it is some kind of lookup table, so that Helix-publish can take the workflow ID to look up the hostname.

**Comment by kptdobe** [comment]
We should use the pipeline number: `<< pipeline.number >>`. See https://circleci.com/docs/2.0/pipeline-variables/ 
It is incremental and can be easily retrieved based on the workflow id via an API call: `https://circleci.com/api/v2/workflow/{workflow_id}` gives you `pipeline_number`. See https://circleci.com/api/v2/workflow/e8c76096-09a8-45df-b601-737ddc1c5e66
